temp_target: 40.0
temp:
  - real_model:
    - TaskZone:
      - increase: 4.1      # Celcius Degree per second
      - decrease: 5.1    # Celcius Degree per second
    - HighLocZone:
      - increase: 4.2      # Celcius Degree per second
      - decrease: 5.2    # Celcius Degree per second
    - ColdZone:
      - increase: 4.3      # Celcius Degree per second
      - decrease: 5.3    # Celcius Degree per second
  - prediction_model:
    - TaskZone:
      - increase: 4.4      # Celcius Degree per second
      - decrease: 5.4    # Celcius Degree per second
    - HighLocZone:
      - increase: 4.5      # Celcius Degree per second
      - decrease: 5.5    # Celcius Degree per second
    - ColdZone:
      - increase: 4.6      # Celcius Degree per second
      - decrease: 5.6    # Celcius Degree per second

localization_target: 0.5
localization:
  - real_model:
    - TaskZone:
      - loc_uncertainty: 1.1      # Meters
    - HighLocZone:
      - loc_uncertainty: 1.2      # Meters
    - ColdZone:
      - loc_uncertainty: 1.3      # Meters
  - prediction_model:
    - TaskZone:
      - loc_uncertainty: 1.4      # Meters
    - HighLocZone:
      - loc_uncertainty: 1.5      # Meters
    - ColdZone:
      - loc_uncertainty: 1.6      # Meters

renewal_time:
  - real_model:
    - fromTaskZone:
      - toTaskZone: 0.01
      - toHighLocZone: 2.0
      - toColdZone: 2.0
    - fromHighLocZone:
      - toTaskZone: 2.0
      - toHighLocZone: 0.01
      - toColdZone: 1.0
    - fromColdZone:
      - toTaskZone: 2.0
      - toHighLocZone: 1.0
      - toColdZone: 0.01
  - prediction_model:
    - fromTaskZone:
      - toTaskZone: 0.01
      - toHighLocZone: 2.0
      - toColdZone: 2.0
    - fromHighLocZone:
      - toTaskZone: 2.0
      - toHighLocZone: 0.01
      - toColdZone: 1.0
    - fromColdZone:
      - toTaskZone: 2.0
      - toHighLocZone: 1.0
      - toColdZone: 0.01


arrival_task_per_second: 1.5
task_metrics:
  - real_model:
    - TaskZone:
      - departing_tasks_rate: 1.1      # Tasks per second
    - HighLocZone:
      - departing_tasks_rate: 1.2      # Tasks per second
    - ColdZone:
      - departing_tasks_rate: 1.3      # Tasks per second
  - prediction_model:
    - TaskZone:
      - departing_tasks_rate: 1.4      # Tasks per second
    - HighLocZone:
      - departing_tasks_rate: 1.5      # Tasks per second
    - ColdZone:
      - departing_tasks_rate: 1.6      # Tasks per second
